test/end-to-end/passwordless.test.js

```diff
@@ -1317,6 +1369,29 @@ export function getPasswordlessTestCases({ authRecipe, logId, generalErrorRecipe
                 ]);
             });
 
+            it("Successful signin w/ stored redirectToPath (only fragment) and email verification", async function () {
```

Comment:
Isn't this a duplicate? What am I missing?

Reply:
This for "links" and the other one is for code.